commercial/app/controllers/commercial/package.scala

```diff
@@ -20,7 +20,7 @@ package object commercial {
   }
 
   def specificId(implicit request: RequestHeader): Option[String] = request.queryString.get("t").map(_.head)
-  def specificIds(implicit request: RequestHeader): Seq[String] = request.queryString.getOrElse("t", Nil).reverse
```

Comment:
@kenlim do you know why you chose to reverse this list [originally](https://github.com/guardian/frontend/commit/0793d4ee905bd7dfc34855cd65c229b8aa99fa43#diff-80eb57197976cd6ca6bb91adc994aee3R19)?